src/SablierV2NFTDescriptor.sol

```diff
@@ -78,6 +78,13 @@ contract SablierV2NFTDescriptor is ISablierV2NFTDescriptor {
             })
         );
 
+        // Performs a low-level call to handle v2.0 contracts in which `isTransferable` is not implemented.
+        (bool success, bytes memory returnData) =
+            address(vars.sablier).staticcall(abi.encodeWithSignature("isTransferable(uint256)", streamId));
```

Comment:
There's a nicer way to do this with [`abi.encodeCall`](https://twitter.com/PaulRBerg/status/1494307965729480704).